documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -135,8 +135,21 @@ spec:
 EOF
 ----
 <1> Specify the name of the VMware `Provider` CR.
-<2> Specify the managed object reference (MOR) of the VMware host.
+<2> Specify the managed object reference (MOR) of the VMware host. To retrieve the name of the MOR, see the note that follows.
 <3> Specify the IP address of the VMware migration network.
++
+[NOTE]
+====
+To retrieve the name of the MOR:
```

Comment:
not sure about "the name of the MOR" since it's not really its name, its the MOR itself
maybe just "To retrieve the MOR of an entity:" ?

Reply:
Fixed.